core/templates/services/date-time-format.service.ts

```diff
@@ -25,10 +25,18 @@ import moment from 'moment';
   providedIn: 'root'
 })
 export class DateTimeFormatService {
-  // Returns just the time if the local datetime representation has the
-  // same date as the current date. Otherwise, returns just the date if the
-  // local datetime representation has the same year as the current date.
-  // Otherwise, returns the full date (with the year abbreviated).
+/**
+  * This function returns just the time if the local datetime representation
```

Comment:
Remove the word `just`. Also, can you frame it like `The function returns X if Y. Else if A, it returns B. Else, it returns C`?

Reply:
Done